lib/management/configuration.js

```diff
@@ -149,6 +149,15 @@ function patchConfiguration(newConf, log, cb) {
                 case 'location-gcp-v1':
                     location.type = 'gcp';
                     if (l.details.secretKey && l.details.secretKey.length > 0) {
+                        let https = true;
```

Comment:
We can just set `https: true` instead of this logic. It was implemented for `aws_s3` type because we wanted to supported HTTP only endpoint for S3 Connector while using https for AWS.